data_summaries/Open-Platypus.json

```diff
@@ -0,0 +1,556 @@
+            "Language Style Analysis",
+            "Multiple Choice Question Answering",
+            "Natural Language Inference",
+            "Question Answering"
+        ],
+        "Text Sources": [
+            "books",
+            "gmat",
+            "lsat",
+            "undisclosed web"
+        ],
+        "Model Generated": [],
+        "Format": [
+            "Zero-shot"
+        ],
+        "Human Annotation": "Yes",
+        "Derived from Datasets": [],
+        "Creators": [
+            "National University of Singapore"
+        ],
+        "Licenses": [
+            {
+                "License": "Non Commercial",
+                "License URL": "https://whyu.me/reclor/"
+            }
+        ],
+        "License Notes": "Website states dataset is for non-commercial research purposes only.",
+        "License Verified By": "Ariel Lee",
+        "Dataset Filter IDs": [
+            "reclor"
+        ]
+    },
+    "op-scibench": {
+        "Unique Dataset Identifier": "op-scibench",
+        "Dataset Name": "scibench",
+        "Paper Title": "SciBench: Evaluating College-Level Scientific Problem-Solving Abilities of Large Language Models",
+        "Dataset URL": "https://huggingface.co/datasets/xw27/scibench",
+        "GitHub URL": "https://github.com/mandyyyyii/scibench",
+        "Hugging Face URL": "https://huggingface.co/datasets/xw27/scibench",
+        "Papers with Code URL": "https://paperswithcode.com/dataset/scibench",
+        "ArXiv URL": "https://arxiv.org/abs/2307.10635",
+        "Semantic Scholar Corpus ID": 259991511,
+        "Collection": "Open-Platypus",
+        "Collection URL": "https://platypus-llm.github.io",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "Logical and Mathematical Reasoning",
+            "Question Answering"
+        ],
+        "Text Sources": [
+            "calculus: early transcendentals",
+            "classical dynamics of particles and systems",
+            "elementary differential equations and boundary value problems",
+            "fundamentals of physics",
+            "physical chemistry",
+            "physical chemistry, quanta, matter, and change",
+            "probability and statistical inference",
+            "quantum chemistry",
+            "statistical thermodynamics"
+        ],
+        "Model Generated": [],
+        "Format": [
+            "Chain-of-Thought",
+            "Few-shot",
+            "Zero-shot"
+        ],
+        "Human Annotation": "Yes",
+        "Derived from Datasets": [],
+        "Creators": [
+            "UCLA",
+            "University of Washington"
+        ],
+        "Licenses": [
+            {
+                "License": "MIT License",
+                "License URL": "https://github.com/mandyyyyii/scibench/tree/main"
+            }
+        ],
+        "License Notes": "The license on GitHub is MIT and no license is provided on HF or in the paper.",
```

Comment:
In that case let's put Unspecified. We can programmatically grab the HF/GH licenses later to compare, but we try to only document confirmed data licenses here.